app/assets/stylesheets/helpers/_old-ie.scss

```diff
@@ -0,0 +1,42 @@
+// static specific styles
+#wrapper,
+.survey-wrapper,
+.govuk-emergency-banner div {
+  width: 960px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.js-enabled .govuk-sticky-element--hidden {
+  // XXX: Would be nice to `@include .visuallyhidden`.
+  position: absolute;
+  left: -9999em;
+}
+
+// govuk-frontend style reinforcing
+// needs to be redefined for old IE as govuk-frontend media queries confuse it
+// hence also the additional nesting in the body element
+body {
+  .govuk-width-container {
+    width: 960px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .govuk-grid-row {
+    margin: 0 -15px;
+    clear: both;
+  }
+
+  .govuk-grid-column-one-third {
+    float: left;
+    width: 33.3333%;
+    padding: 0 15px;
+  }
+
+  .govuk-grid-column-two-thirds {
+    float: left;
+    width: 66.6666%;
+    padding: 0 15px;
+  }
+}
```

Comment:
I'd suggest that this code for frontend should be moved to the frontend app to stop every app needing to download and parse these styles. 

Moving it to [frontend's application-ie-8.scss](https://github.com/alphagov/frontend/blob/master/app/assets/stylesheets/application-ie8.scss) could also remove some duplication, as there is overlap between this and frontend's IE8 specific stylesheet. 

For example, frontend's IE8 stylesheet has this as well:
```scss
.govuk-grid-column-two-thirds {
  float: left;
  width: 66.6666%;
  padding: 0 15px;
}
```

Reply:
😄 agreed. This whole PR actually came about partly because I'd already written these styles in `frontend` and then realised if we need them there we need them everywhere. Will raise that PR once I've finished with this one 👍

Reply:
Also massive thanks for the review @injms !!